includes/class-sensei-course.php

```diff
@@ -2825,17 +2825,19 @@ public static function the_course_lessons_title() {
 	}//end the_course_lessons_title()
 
 	/**
-	 * This function loads the global wp_query object with with lessons
-	 * of the current course. It is designed to be used on the single-course template
+	 * Get a wp_query object with with lessons of the current course.
+	 * It is designed to be used on the single-course template
 	 * and expects the global post to be a singular course.
 	 *
 	 * This function excludes lessons belonging to modules as they are
 	 * queried separately.
 	 *
 	 * @since 1.9.0
-	 * @global $wp_query
+	 * @since 3.0.0 Returns the WP_Query object.
+	 *
+	 * @return $wp_query
 	 */
-	public static function load_single_course_lessons_query() {
```

Comment:
I think that we should deprecate `load_single_course_lessons_query` and add `get_single_course_lessons_query` as a new method.

Reply:
Makes sensei! Fixed here: https://github.com/Automattic/sensei/pull/3032/commits/1c20d4555b45d5f3c8bfd56b65905d2795e860bd